### PR TITLE
Update proto-definition.md

### PR DESCRIPTION
**Pull request by omichel**
Added PROTO header information in the documentation.
See https://cyberbotics.com/doc/reference/proto-definition?version=fix-documentation-added-proto-header

### CODE REVIEW

File: docs/reference/proto-definition.md [hunk 1]
```diff
@@ -22,9 +40,13 @@ field fieldType fieldName defaultValue
...
 - `defaultValue` is a literal default value that depends on `fieldType`.
 
-Here is an example of PROTO definition:
+#### Summary
+
+Here is how a PROTO file looks like:
 
 ```
+#VRML_SIM {{ webots.version.major }} utf8
+
```
Comment (DavidMansolino) [inline docs/reference/proto-definition.md:49]: Maybe we can add the comment example here too.

Comment (omichel) [reply docs/reference/proto-definition.md:49]: I just wanted to keep it minimal here, given that these comments are not mandatory, unlike the header line.

Comment (omichel) [reply docs/reference/proto-definition.md:49]: A more complete example is available later in this chapter.

**Review by DavidMansolino — APPROVED** [review]
Thank you.

**Pull request merged**